cypress/e2e/filter/filterChips.cy.ts

```diff
@@ -54,8 +58,7 @@ describe("Case list", () => {
         cy.get(".moj-filter__tag").contains("Exceptions").should("not.exist")
 
         // Removes the filter chip
```

Comment:
I think we can remove this comment, function name is self descreptive

Reply:
Comment removed